app/config/services/membership.xml

```diff
@@ -45,5 +45,16 @@
             <tag name="form.type" />
         </service>
 
+        <service id="app.membership.membership_choose_nearby_committee_type" class="AppBundle\Form\MembershipChooseNearbyCommitteeType">
+            <argument type="service" id="doctrine.orm.entity_manager"/>
+            <argument type="service" id="bazinga_geocoder.geocoder"/>
```

Comment:
This argument is not used in your class. Thus, we should not rely on `bazinga_geocoder.geocoder` service but `app.geocoder` service instead, which is a wrapper with a custom interface.